pkg/testutils/lint/lint_test.go

```diff
@@ -60,6 +60,10 @@ var (
 // the License. You may obtain a copy of the License at
 //
 //     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+`)
+	cslHeader = regexp.MustCompile(`// Copyright 20\d\d The Cockroach Authors.
+// Use of this software is governed by the CockroachDB Software License
```

Comment:
This [and other] backports of https://github.com/cockroachdb/cockroach/pull/131665 should incorporate the fix from https://github.com/cockroachdb/cockroach/pull/131709/files

Reply:
Indeed. I've made the change to this PR and will do the same for the other PRs when I backport the changes.